launch/tier4_localization_launch/launch/localization.launch.xml

```diff
@@ -21,22 +24,31 @@
         <arg name="input/pointcloud" value="$(var input/pointcloud)"/>
         <arg name="use_pointcloud_container" value="$(var use_pointcloud_container)"/>
         <arg name="pointcloud_container_name" value="$(var pointcloud_container_name)"/>
+        <arg name="localization_mode" value="$(var localization_mode)"/>
       </include>
     </group>
     <!-- pose_estimator module -->
     <group>
       <push-ros-namespace namespace="pose_estimator"/>
-      <include file="$(find-pkg-share tier4_localization_launch)/launch/pose_estimator/pose_estimator.launch.xml"/>
+      <include file="$(find-pkg-share tier4_localization_launch)/launch/pose_estimator/pose_estimator.launch.xml">
+        <arg name="localization_mode" value="$(var localization_mode)"/>
+      </include>
     </group>
     <!-- twist_estimator module -->
     <group>
       <push-ros-namespace namespace="twist_estimator"/>
-      <include file="$(find-pkg-share tier4_localization_launch)/launch/twist_estimator/twist_estimator.launch.xml"/>
+      <include file="$(find-pkg-share tier4_localization_launch)/launch/twist_estimator/twist_estimator.launch.xml">
+        <arg name="localization_mode" value="$(var localization_mode)"/>
+        <arg name="use_eagleye_twist" value="$(var use_eagleye_twist)"/>
+      </include>
     </group>
     <!-- pose_twist_fusion_filter module -->
     <group>
       <push-ros-namespace namespace="pose_twist_fusion_filter"/>
-      <include file="$(find-pkg-share tier4_localization_launch)/launch/pose_twist_fusion_filter/pose_twist_fusion_filter.launch.xml"/>
+      <include file="$(find-pkg-share tier4_localization_launch)/launch/pose_twist_fusion_filter/pose_twist_fusion_filter.launch.xml">
+        <arg name="localization_mode" value="$(var localization_mode)"/>
+        <arg name="use_eagleye_twist" value="$(var use_eagleye_twist)"/>
```

Comment:
This is unnecessary because pose_twist_fusion_filterdoes not need to be aware of the input source.